web-app/CKS.Web.Test/UnitTests/Controllers/SearchControllerTests.cs

```diff
@@ -1,23 +1,65 @@
-using System;
-using System.Collections.Generic;
-using System.Text;
 using Shouldly;
 using Xunit;
 using CKS.Web.Controllers;
 using Moq;
 using Microsoft.Extensions.Logging;
+using NICE.Search.Common.Urls;
+using NICE.Search.Common.Interfaces;
+using NICE.Search.Common.Models;
+using System;
+using CKS.Web.Test.IntegrationTests.Infrastructure.Extensions;
 
 namespace CKS.Web.Test.UnitTests.Controllers
 {
 	public class SearchControllerTests
 	{
-		[Obsolete("Please replace this test with a real test when the search endpoint is setup")]
 		[Fact]
-		public void ASampleTestReturnsNonNullObject()
+		public void ASearchCanBePerformed()
```

Comment:
I think this test name could probably be a bit more explicit as it's probably not clear what performing a search means. Get data back? 200 response? correct data?